packages/admin/admin-rte/src/core/Controls/BlockTypesControls.tsx

```diff
@@ -32,49 +91,11 @@ function BlockTypesControls({ disabled, blockTypes, classes }: Props & WithStyle
                     </MenuItem>
                 ))}
             </Select>
-        </FormControl>
+        </Root>
     );
 }
 
-export type RteBlockTypeControlsClassKey = "root" | "select";
-
-const styles = (theme: Theme) => {
-    const rteTheme = getRteTheme(theme.components?.CometAdminRte?.defaultProps);
-
-    return createStyles<RteBlockTypeControlsClassKey, Props>({
-        root: {
-            [`& .${inputBaseClasses.root}`]: {
-                backgroundColor: "transparent",
-                height: "auto",
-                border: "none",
-                "&, &:hover": {
-                    "&:before, &:after": {
-                        borderBottomWidth: 0,
-                    },
-                },
-            },
-            [`& .${selectClasses.icon}`]: {
-                top: "auto",
-                color: "inherit",
-            },
-        },
-        select: {
-            color: rteTheme.colors.buttonIcon,
-            minWidth: 180,
-            lineHeight: "24px",
-            fontSize: 14,
-            padding: 0,
-
-            [`&.${selectClasses.select}`]: {
-                minHeight: 0,
-            },
-        },
-    });
-};
-
-const StyledBlockTypesControls = withStyles(styles, { name: "CometAdminRteBlockTypeControls" })(BlockTypesControls);
-
-// If there are no dropdown-features, this must return null not just an empty component, to prevent an empty item from being rendered in Toolbar
+//If there are no dropdown-features, this must return null not just an empty component, to prevent an empty item from being rendered in Toolbar
```

Comment:
```suggestion
// If there are no dropdown-features, this must return null not just an empty component, to prevent an empty item from being rendered in Toolbar
```